core/src/test/java/org/infinispan/jmx/ComponentsJmxRegistrationTest.java

```diff
@@ -48,13 +48,6 @@ public void tearDown() {
       cacheContainers.clear();
    }
 
-   public void testStopStartCM() throws Exception {
```

Comment:
Does this test not work anymore? Do we disallow restarting a stopped `EmbeddedCacheManager`? If so we should make sure to document and add a simple test like this that asserts an exception etc.

Reply:
Re-enabled it